Update public_test_suites.md (#2593)
buildkite/docs: pages/test_analytics/public_test_suites.md

# Public test suites

> Test suites can now be made public. Perhaps you're working on an open-source project and want the whole world to be able to see your test analytics, or perhaps you're proud of recent optimizations you've made to improve the health and reliability of your test suite and need to share the results of your work.

Reviewer: I think this skews too far on the marketing and changelog style. Time words like "now" don't generally age well in the docs. Perhaps:

```suggestion
If you're working on an open-source project or just want to share your test suite analytics with the world, you can make your test suite public.
```
